src/Protocol/ActivityPub/ClientToServer.php

```diff
@@ -323,7 +323,7 @@ public static function getOutbox(array $owner, int $uid, int $page = null, int $
 			$requester_id = Contact::getIdForURL($requester, $owner['uid']);
 			if (!empty($requester_id)) {
 				$permissionSets = DI::permissionSet()->selectByContactId($requester_id, $owner['uid']);
-				if (!empty($permissionSets)) {
+				if ($permissionSets->getTotalCount() > 0) {
```

Comment:
Total count is only meant to be used with paginated result sets. Here, a simple `count()` should be used instead. The Collection extends ArrayIterator which extends Countable as well. Please test first though, don't take my word for it.
```suggestion
				if (count($permissionSets)) {
```

Reply:
You are right. I havn't thought about the ArrayIterator. :+1: